controllers/servers/host_definer/watcher/secret_watcher.py

```diff
@@ -0,0 +1,47 @@
+from kubernetes import watch
+
+from controllers.servers.config import SECRET_ARRAY_PARAMETER
+from controllers.common.csi_logger import get_stdout_logger
+from controllers.servers.host_definer.watcher.watcher_helper import Watcher, SECRET_IDS
+from controllers.servers.host_definer.common import settings
+
+logger = get_stdout_logger()
+
+
+class SecretWatcher(Watcher):
+
+    def watch_secret_resources(self):
+        while True:
+            resource_version = self.core_api.list_secret_for_all_namespaces().metadata.resource_version
+            stream = watch.Watch().stream(self.core_api.list_secret_for_all_namespaces,
+                                          resource_version=resource_version, timeout_seconds=5)
+            for event in stream:
+                if event[settings.TYPE_KEY] == settings.ADDED_EVENT and (
+                        self._is_secret_used_by_storage_class(event)):
+                    self._verify_host_defined_from_secret_event(event)
+                if event[settings.TYPE_KEY] == settings.MODIFIED_EVENT:
+                    if self._is_secret_used_by_storage_class(event):
+                        self._handle_modified_secrets(event)
```

Comment:
if I understand correctly
```suggestion
               if self._is_secret_used_by_storage_class(event)):
                   event = event[settings.TYPE_KEY]
                    if event == settings.ADDED_EVENT:
                        self._verify_host_defined_from_secret_event(event)
                    if event == settings.MODIFIED_EVENT:
                        self._handle_modified_secrets(event)
```